en/configure-a-tidb-cluster.md

```diff
@@ -1,35 +1,252 @@
 ---
-title: Resource and Disaster Recovery Configurations
-summary: Learn the resource and disaster recovery configurations of a TiDB cluster in Kubernetes.
-category: reference
+title: Configure a TiDB Cluster in Kubernetes
+summary: Learn how to configure a TiDB cluster in Kubernetes.
+category: how-to
 ---
 
-# Resource and Disaster Recovery Configurations
+This document introduces how to configure a TiDB cluster for production deployment. It covers the following content:
 
-This document introduces the following items of a TiDB cluster in Kubernetes:
+- Resource configuration
 
-+ The configuration of resources
-+ The configuration of disaster recovery
+- Configure a TiDB deployment
```

Comment:
Also need to include the configuration for services, e.g. the configures [here](https://pingcap.com/docs-cn/tidb-in-kubernetes/stable/access-tidb/)

Reply:
Can I address this in a follow up PR?